src/app/compose/emailvalidator.ts

```diff
@@ -35,3 +35,14 @@ export function isValidEmail(email: string) {
         return EMAIL_REGEXP.test(email);
     }
 }
+
+/**
+ * Validate each email in the list, return true if all are valid
+ * @param emailList
+ */
+export function isValidEmailList(emailList: string) {
+  if (!emailList) {
+    return false;
+  }
+  return emailList.split(',').every((recipient => isValidEmail(recipient)));
+}
```

Comment:
This seems to be using 2 spaces for indentation while the rest of the file uses 4 :)